-------------------------------------------------------------------------------
Test set: pro.abacus.javatutor.repository.UserRepositoryTest
-------------------------------------------------------------------------------
Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 64.206 s <<< FAILURE! - in pro.abacus.javatutor.repository.UserRepositoryTest
shouldLookUpSavedAccount(pro.abacus.javatutor.repository.UserRepositoryTest)  Time elapsed: 56.396 s  <<< ERROR!
org.springframework.dao.DataAccessResourceFailureException: Timed out after 30000 ms while waiting to connect. Client view of cluster state is {type=UNKNOWN, servers=[{address=localhost:27017, type=UNKNOWN, state=CONNECTING, exception={com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.ConnectException: Connection refused: connect}}]; nested exception is com.mongodb.MongoTimeoutException: Timed out after 30000 ms while waiting to connect. Client view of cluster state is {type=UNKNOWN, servers=[{address=localhost:27017, type=UNKNOWN, state=CONNECTING, exception={com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.ConnectException: Connection refused: connect}}]
	at pro.abacus.javatutor.repository.UserRepositoryTest.shouldLookUpSavedAccount(UserRepositoryTest.java:50)
Caused by: com.mongodb.MongoTimeoutException: Timed out after 30000 ms while waiting to connect. Client view of cluster state is {type=UNKNOWN, servers=[{address=localhost:27017, type=UNKNOWN, state=CONNECTING, exception={com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.ConnectException: Connection refused: connect}}]
	at pro.abacus.javatutor.repository.UserRepositoryTest.shouldLookUpSavedAccount(UserRepositoryTest.java:50)

shouldLookUpSavedAccount(pro.abacus.javatutor.repository.UserRepositoryTest)  Time elapsed: 56.396 s  <<< ERROR!
com.mongodb.MongoTimeoutException: Timed out after 30000 ms while waiting to connect. Client view of cluster state is {type=UNKNOWN, servers=[{address=localhost:27017, type=UNKNOWN, state=CONNECTING, exception={com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.ConnectException: Connection refused: connect}}]
	at pro.abacus.javatutor.repository.UserRepositoryTest.tearDown(UserRepositoryTest.java:45)

